doc/langdef.md

```diff
@@ -40,7 +40,7 @@ Member         = Primary
                | Member "[" Expr "]"
                | Member "{" [FieldInits] "}"
                ;
-Primary        = ["."] IDENT ["(" [ExprList] ")"]
+Primary        = IDENT ["(" [ExprList] ")"]
```

Comment:
The optional leading `.` isn't covered by the `Member` rule and is used to disambiguate type resolution from relative (no leading dot) to global (leading dot). This is the same as proto semantics for name resolution.

Reply:
i see, i initially didn't understand the need for this. I will have to give the protobuf docs another look. but i can remove this in the revision this pr

Reply:
it maybe helpful to add your explanation about global type resolution somewhere in the grammer doc, just so it's explicit

Reply:
I think it's mentioned in the spec, but we can update the grammar as well.